src/main/java/net/minecraftforge/common/JigsawCategory.java

```diff
@@ -0,0 +1,145 @@
+package net.minecraftforge.common;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import com.mojang.datafixers.util.Pair;
```

Comment:
Please use the apache pair everywhere where vanilla does not require a datafixer pair, as this is what the rest of the codebase uses as well